src/libs/vue-h-zoom.js

```diff
@@ -32,6 +40,10 @@ export default {
     zoomWindowY: {
       type: Number,
       default: 10
+    },
+    backgroundOption: {
+      type: Object,
+      default : () => DEFAULT_BACKGROUND_OPTIONS
```

Comment:
let's avoid giving this a default value.. make new computed, returning this object when value of backgroundOptions is falsy

Reply:
are we going to allow users to specify only some options pak? 
such as, if users only want to change color then they only specify `color` in the prop and not all 5 properties
```html
<vue-h-zoom background-options="{color: 'blue'}" />
```